src/wp-includes/rest-api/endpoints/class-wp-rest-attachments-controller.php

```diff
@@ -135,6 +135,32 @@ public function create_item_permissions_check( $request ) {
 			);
 		}
 
+		/**
+		 * Filter whether the server should prevent uploads for image types it doesn't support. Default true.
+		 *
+		 * Developers can use this filter to enable uploads of certain image types.
```

Comment:
Missing `@since` tag

Reply:
added in cdb9d8b9f7ee7a53a5f690d85cb8661a8dd9f52a